index/readme.md

```diff
@@ -1,17 +1,55 @@
-# opendatacube/datacube-index
 
+# opendatacube/datacube-index
 Dockerfile for use in indexing into the Open Data Cube
 
+The build images are hosted here: https://hub.docker.com/r/opendatacube/datacube-index.
```

Comment:
Would it be worth also linking back to the github repo? That way there's a way to find it from DockerHub

Reply:
yes